cmd/state-tray/internal/open/open_lin.go

```diff
@@ -2,12 +2,29 @@
 
 package open
 
-import "errors"
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/ActiveState/cli/internal/locale"
+)
 
 func Prompt(command string) error {
-	return errors.New("not implemented")
-}
+	shellData, err := exec.Command("bash", "-c",
+		"grep ^$(id -un): /etc/passwd | cut -d: -f7-",
+	).Output()
+	if err != nil {
+		return locale.WrapError(err,
+			"err_determine_default_shell", "Could not determine default shell",
+		)
+	}
+	shell := string(shellData[:len(shellData)-1]) // trim newline
```

Comment:
What, Bash couldn't do this? :p

Reply:
Why, yes. Yes, it can.

Reply:
This is resolved by the resolution of another thread.